src/index.js

```diff
@@ -1,7 +1,64 @@
-require('file-loader?name=[name].[ext]!./index.html');
+require('file-loader?name=[name].[ext]!./front_end/index.html');
+import './style.css';
 
 const heading = document.getElementById("heading");
+var data = "";
+var patientId = "";
 
-if (heading != null) {
-    heading.textContent = "hi";
+patientId = localStorage.getItem("patientID");
+
+function httpGet(theUrl)
+{
+    var xmlHttp = new XMLHttpRequest();
+    xmlHttp.open( "GET", theUrl, false ); // false for synchronous request
+    xmlHttp.send( null );
+    return xmlHttp.responseText;
+}
+
+document.getElementById('test').onClick = getAllData()
+
+function getAllData() {
+    data = httpGet("http://localhost:3000/patients")
+    const d= JSON.parse(data)
+    appendData(d)
+}
+
+    
+function appendData(data) {
+    var mainContainer = document.getElementById("patientData");
+    console.log(mainContainer)
+    //append each person to the page
+    for (var i = 0; i < data.length; i++) {
+        var div = document.createElement("div");
+        div.className = "list-group-item list-group-item-action"
+        div.innerHTML = '<b>Patient ID: </b>' + data[i].person_id + ',<b> Family name: </b>' + data[i].family_name;
```

Comment:
verwerk alle info die je krijgt

Reply:
Nee, niet noodzakelijk op hoofdscherm

Reply:
gaat hier weer om die prefix en suffix dingen

Reply:
is overbodig